download.py

```diff
@@ -20,7 +20,7 @@
 from os import path
 
 IFRAME_CSS_SELECTOR = '.iframe-container>iframe'
-
+POPUP_ACCEPT_CSS_SELECTOR = '#qc-cmp2-ui > div.qc-cmp2-footer.qc-cmp2-footer-overlay.qc-cmp2-footer-scrolled > div > button.sc-ifAKCX.ljEJIv'
```

Comment:
This selector is based on auto generated classname of styled component, so it's pretty unstable way of detecting "AGREE" button.
You better try something like this:
```python
driver.find_element_by_xpath("//button[contains(text(),'AGREE')]").click()
```

Reply:
Changed it to xpath.